.github/workflows/build.yaml

```diff
@@ -957,6 +1100,18 @@ jobs:
           podman run --rm --volume ${PODMAN_SOCK}:/var/run/docker.sock docker.io/anchore/grype:latest \
             --only-fixed --fail-on low ${{ env.ImageName }}
 
+  container_arm64:
+    name: arm64 container image
+    runs-on: self-hosted
```

Comment:
What do you think of using a more specific name, like: `self-hosted-arm64` (just in case we add
new self hosted runners for other platforms)?

Reply:
> What do you think of using a more specific name, like: `self-hosted-arm64` (just in case we add new self hosted runners for other platforms)?

I have added a new label called self-hosted-arm64  to the Rasperrry Pi 5 hosted runner - 
https://github.com/skupperproject/skupper-router/settings/actions/runners/25

I will change one single line in build.yml from self-hosted to self-hosted-arm64 and if it works, I will change the rest to self-hosted-arm. Stay tuned !

Reply:
> It worked !! I will change all self-hosted to self-hosted-arm64

Done !